provider/aws/dist/rack.json

```diff
@@ -1680,16 +1680,16 @@
       },
       "Type": "AWS::EC2::SecurityGroup"
     },
-    "RackWeb": {
-      "DependsOn": [ "RackWebTasks", "Balancer", "Cluster", "CustomTopic", "Instances", "ServiceRole" ],
+    "WebService": {
```

Comment:
In the docker-compose.yml for Rack these are `web` and `monitor`. Eventually I'd like Rack's own cloudformation template to come from the same code that generates our templates for apps.